packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -28,6 +31,28 @@ module.exports = (log, db, config, customs, push, oauthdb, subhub) => {
   }
 
   return [
+    {
+      method: 'GET',
+      path: '/oauth/subscriptions/clients',
+      options: {
+        auth: {
+          payload: false,
+          strategy: 'subscriptionsSecret',
+        },
+        response: {
+          schema: isA.array().items(
+            isA.object().keys({
+              client_id: isA.string(),
```

Comment:
Eek no, it's not, `service` there is the human-readable name (provided it's set in config). I see your point, but rather than change this to `service` I wonder if we should add `client_id` to those notification structures.

That human-readable name is set in config purely for metrics purposes (it's what shows up in Amplitude), and may be liable to change over time. Client id is more reliable.

Reply:
Adding `client_id` to the notifications makes sense to me.

Reply:
Ref: #1210